egs/librispeech/ASR/transducer/train.py

```diff
@@ -592,6 +595,23 @@ def run(rank, world_size, args):
     if params.full_libri:
         train_cuts += librispeech.train_clean_360_cuts()
         train_cuts += librispeech.train_other_500_cuts()
+
+    def remove_short_and_long_utt(c: Cut):
+        # Keep only utterances with duration between 1 second and 20 seconds
+        return 1.0 <= c.duration <= 20.0
```

Comment:
After removing very long utterances, it is able to double the value of max-duration, e.g., from 150 to 300.

```
2021-12-13 16:33:29,079 INFO [train.py:608] (0/4) Before removing short and long utterances: 843723
2021-12-13 16:33:29,079 INFO [train.py:609] (0/4) After removing short and long utterances: 843526
2021-12-13 16:33:29,079 INFO [train.py:610] (0/4) Removed 197 utterances (0.02335%)
```

Maybe we can try it in `conformer_ctc/train.py` and see how it affects the WER.
I believe it will reduce the training time as it allows us to use a larger batch size.